ui/app/send.js

```diff
@@ -239,6 +317,7 @@ SendTransactionScreen.prototype.onSubmit = function () {
   var txParams = {
     from: this.props.address,
     value: '0x' + value.toString(16),
+    gasMultiplier: gasMultiplier * 0.01,
```

Comment:
if `gasMultiplier` is a string, since it's fresh off an `input.value`, what will multiplying it by 0.01 do?